### PR TITLE
[Accel] Add cohort selector on insturctor dashboard

### PR DESCRIPTION
**Pull request by flying-pi**
Added cohort select into the ORA result page.

Task: https://youtrack.raccoongang.com/issue/ACCEL-443

### CODE REVIEW

File: openassessment/data.py [hunk 1]
```diff
@@ -524,7 +527,20 @@ def collect_ora2_responses(cls, course_id, desired_statuses=None):
...
-        items = AssessmentWorkflow.objects.filter(course_id=course_id, status__in=statuses).values('item_id', 'status')
+        items_qs = AssessmentWorkflow.objects.filter(course_id=course_id, status__in=statuses)
+        if selected_cohort:
+            anonymous_user_ids = (CohortMembership.objects.
+                  filter(course_id=CourseKey.from_string(course_id), course_user_group__id=selected_cohort).
+                  values('user__anonymoususerid__anonymous_user_id')
+            )
+            relevant_submissions_uuid = (Submission.objects.
+                filter(student_item__student_id__in=anonymous_user_ids).
+                values('uuid')
```
Comment (arsentur) [inline openassessment/data.py]: We can use `.values_list('uuid', flat=True)`. In this case `relevant_submissions_uuid_list` on L540 looks redundant.

File: openassessment/data.py [hunk 2]
```diff
@@ -524,7 +527,20 @@ def collect_ora2_responses(cls, course_id, desired_statuses=None):
...
+            )
+            relevant_submissions_uuid_list = [str(uuid['uuid']) for uuid in relevant_submissions_uuid]
+            items_qs = items_qs.filter(submission_uuid__in = relevant_submissions_uuid_list)
```
Comment (arsentur) [inline openassessment/data.py]: Please, remove spaces in filter.

File: openassessment/xblock/openassessmentblock.py [hunk 3]
```diff
@@ -445,7 +444,8 @@ def ora_blocks_listing_view(self, context=None):
         ora_item_view_enabled = context.get('ora_item_view_enabled', False) if context else False
         context_dict = {
             "ora_items": json.dumps(ora_items),
-            "ora_item_view_enabled": ora_item_view_enabled
+            "ora_item_view_enabled": ora_item_view_enabled,
+            "cohort_info": context.get('cohort_info', [])
```
Comment (arsentur) [inline openassessment/xblock/openassessmentblock.py]: Missing trailing comma

Comment (flying-pi) [reply openassessment/xblock/openassessmentblock.py]: Thanks, fixed.

File: openassessment/data.py (hunk 2 above)
Comment (flying-pi) [reply openassessment/data.py]: Done

**Pull request merged**